apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1.IntegrationTests/DatastoreFixture.cs

```diff
@@ -88,13 +90,15 @@ public async Task RetryQueryAsync(Func<Task> check)
             }
         }
 
-        public DatastoreDb CreateDatastoreDb(string namespaceId = null)
+        public DatastoreDb CreateDatastoreDb(string namespaceId = null, string databaseId = null)
```

Comment:
Can we delete the whole databases at the end, instead of just deleting entities? (Both for snippets and integration tests.)

Reply:
I'll test this locally once I'll get Noel exception and is able to execute tests via VS.

Reply:
Done.